sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -528,13 +528,15 @@ def test_radius_neighbors_classifier_outlier_labeling():
     def check_array_exception():
         clf = RNC(radius=1, outlier_label=[[5]])
         clf.fit(X, y)
+
```

Comment:
Please avoid changing the style of the code far away from the main changes of your PR to avoid making the diff harder to read than necessary (and minimize the likelihood of conflicts between concurrently merged PRs).

Reply:
Yes, I've just seen it. Reverted in `7aa2c3c`.